src/prelude.rs

```diff
@@ -20,13 +20,12 @@ pub use parking_lot::{Mutex, RwLock};
 
 #[cfg(feature = "client")]
 pub use crate::client::{Client, ClientError, Context, EventHandler, RawEventHandler};
+pub use typemap_rev::{TypeMap, TypeMapKey};
```

Comment:
```suggestion
#[cfg(feature = "client")]
pub use typemap_rev::{TypeMap, TypeMapKey};
```
Feature gates apply on only one item succeeding them.